src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Views.ts

```diff
@@ -430,6 +464,9 @@ export var userProfileDirective = (
                 adhHttp.resolve(scope.path)
                     .then((res) => {
                         scope.userBasic = res.data[SIUserBasic.nick];
+                        getBadges(adhHttp, $q)(res).then((assignments) => {
+                            scope.assignments = assignments;
+                        });
```

Comment:
I think this should be named `badges`, not `assignments`. I think we should rename it here and also in the other places where `getBadges()` is used.